help/search/_posts/2015-02-09-search-syntax.md

```diff
@@ -115,6 +115,33 @@ Synoym expansion also applies to author names, which provide a way to account fo
 
 As a general rule we recommend to use the full name of the person for author searches since as can be seen above the matching rules in ADS are designed to find the maximal set of records consistent with the author specification given by the user.  Rather than disabling the name-variation algorithm described above, we recommend performing refinement of search results via the user interface filters for author names as described in the ["Filter your search" section]({{ site.baseurl }}/help/search/filter).
 
+### Astronomical Objects and Position Search
+
+The query modifier `object:` in fielded searches allows users to search the literature for bibliographic records that have been tagged with astronomical objects by SIMBAD and NED, or for a specified position on the sky ("cone search"). The cone search also makes use of services offered by SIMBAD and NED. 
+
+Queries for astronomical objects via `object:` queries return publications that have been tagged with the canonical names for these objects. This tagging has been done by the SIMBAD and NED teams. Additionally, the object names (as specified in the query) are also used in an ADS query against abstracts, titles and keywords in the astronomy collection (with synonym replacement switched off). The result set is a combination of all these matches. Object queries that target just one service (e.g. SIMBAD) are not currently supported. The `data` facet can be used to filter the results set.
+
+The syntax for cone searches is: 
+
+```
+  object:"RA ±Dec:radius"
+```
+
+where RA and Dec are right ascention and declination J2000 positions, expressed in decimal degrees or in sexagesimal notation (hours minutes seconds and degrees arcmin and arcsec). The plus or minus sign before the declination is ***mandatory***. The search radius may be given in arcmin, decimal or sexagesimal degrees (The default search radius is 2', and the maximum is 60'). Examples:
+
+* A 10' radius may be written as 0.1667 or 0 10
```

Comment:
Can you add an example here using the radius along with the input coordinates? It seems self-explanatory to me, but I know someone will ask otherwise.

Reply:
I've added an example (see below)